### PR TITLE
Add binary file structure page

### PR DESCRIPTION
**Pull request by patrickelectric**

### CODE REVIEW

**Comment by kklemens** [comment]
@patrickelectric Looks good to me. Is there an example of what the expected output is supposed to look like?

File: docs/binary-structure.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+> Note: The binary file structure is done in **big-endian**.
+
+Arrays (**Strings**) are defined with the following structure:
+- Array size (**uint32**)
+- Array data (**byte_array[]**)
+
+The source code that defines this structure can be found in Ping Viewer source code [LogSensorStruct file](https://github.com/bluerobotics/ping-viewer/blob/master/src/link/logsensorstruct.cpp).
+
+## Python example
```
Comment (jaxxzer) [inline docs/binary-structure.md]: suggest committing the example under `/tools` or `/examples` in master, then link it here. (choice is yours)

File: docs/binary-structure.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
+# Binary file structure
```
Comment (jaxxzer) [inline docs/binary-structure.md]: 's/binary flie/binary log file/g'

**Comment by patrickelectric** [comment]
@kklemens added.

**Pull request merged**